walk-in-interview/src/main/java/com/google/job/data/Requirement.java

```diff
@@ -0,0 +1,39 @@
+package com.google.job.data;
+
+import com.google.common.collect.ImmutableMap;
+
+import java.util.Map;
+
+public enum Requirement {
+    // TODO(issue/26): add more requirements
+    O_LEVEL("o-level", ImmutableMap.of("en", "O Level")),
+    ENGLISH("language-english", ImmutableMap.of("en", "English")),
+    DRIVING_LICENSE_C("driving-license-c", ImmutableMap.of("en", "Category C Driving License"));
+
+    private final String requirementId;
+    private final Map<String, String> localizedNameByLanguage;
+
+    Requirement(String requirementId, Map<String, String> localizedNameByLanguage){
+        this.requirementId = requirementId;
+        this.localizedNameByLanguage = localizedNameByLanguage;
+    }
+
+    /**
+     * Gets the id string.
+     *
+     * @return Id string.
```

Comment:
nit:`@return` is superfluous. I recommend this:
```
/** Returns a stable Id, can be used for database storage. */
```

Reply:
Fixed